taxi/models.py

```diff
@@ -0,0 +1,26 @@
+from django.db import models
+from django.contrib.auth.models import AbstractUser

```

Comment:
Use `AUTH_USER_MODEL` instead of `AbstractUser`. According to the checklist, it is best practice to use `AUTH_USER_MODEL` for referencing the user model.